src/app/auth-page/auth-page.component.html

```diff
@@ -0,0 +1,164 @@
+
+<div class="card-wrapper">
+    <div style=" border: 4px solid rgb(219, 218, 218); padding: 1rem; margin-bottom: 1rem;">
+        <ul ngbNav #nav="ngbNav" [(activeId)]="active"  class="nav-tabs">
+            <li [ngbNavItem]='1' class="my-tab" >
+            <a ngbNavLink style="width: 13vw;" 
+                [ngStyle]="{'background-color': active == 1? 'rgb(211, 0, 0)' : 'white', 'color': active==1? 'white':'blue'}"
```

Comment:
Use this to determine if a route is active: https://angular.io/api/router/RouterLinkActive
Don't use plain CSS styling in templates.